### DISCUSSION TITLE
Getting tracks with track IDs with v1 or v2 API

### DISCUSSION THREAD

**Discussion by eric-shim** [tidal-music/discussions#38] [Q&A] [answered]
Hello,

I have been using TIDAL API V1/V2 from the start and I would like to know if there is a way to get multiple tracks with one request.
As far as I know, V1/V2 doesn't have a API for it. I know openapi.tidal.com has it, but I can't use it in our application.

**Comment by stefanmoro** [comment, marked as answer]
Hi @eric-shim,

The APIs you are referring to (api.tidal.com) are not intended for external use, and such it represents an unauthorized usage of TIDALs API. Please see https://github.com/orgs/tidal-music/discussions/7 (and [this comment](https://github.com/orgs/tidal-music/discussions/7#discussioncomment-7124801) in particular) for further information.

The only authorized way to build 3rd party applications on TIDAL is via the [TIDAL Developer Platform](https://developer.tidal.com) (i.e. openapi.tidal.com).

**Reply by eric-shim** [reply]
Thank you .
